packages/ek-components/src/main.js

```diff
@@ -62,6 +64,8 @@ const components = {
   EkSearchBar,
   EkSlidableCardGrid,
   EkSlidableGrid,
+  EkSlidableGridNew,
+  EkSlidableCardGridNew,
```

Comment:
Can we choose a better name for these components? `EkSlidableGrid` vs `EkSlidableGridNew` has zero meaning for someone who was not around when they were created. I tried to think of some suggestions and the best I could come up with was `EkInertialSlidableGrid`, but I'm not sure how good that is.